pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -188,13 +188,17 @@ public void run() {
     return brokerServerBuilder;
   }
 
-  private String getZkAddressForBroker(String zkServers, String helixClusterName) {
+  public static String getZkAddressForBroker(String zkServers, String helixClusterName) {
     List tokens = new ArrayList<String>();
-
-    for (String token : zkServers.split(",")) {
-      tokens.add(StringUtil.join("/", StringUtils.chomp(token, "/"), helixClusterName, PROPERTY_STORE));
+    String[] zkSplit = zkServers.split("/", 2);
```

Comment:
Seems that we need to use propertyStore before HelixManager connected. Will merge this for now.